requirements.txt

```diff
@@ -3,20 +3,18 @@ w3lib==1.13.0
 queuelib==1.4.2
 Scrapy==1.0.5
 
-# Portia
-slybot==0.12.1
-scrapely==0.12.0
```

Comment:
I think we better leave portia and scrapely, this is legacy stack after all and who knows what current spiders imports.